tealeaf/widget/widget.entitlements

```diff
@@ -4,7 +4,7 @@
 <dict>
```

Comment:
looks like same dict is added in file tealeaf/TeaLeafIOS.entitlements as well. Cant we avoid this duplicate?